code-challenges.test.js

```diff
@@ -42,13 +77,40 @@ var cubeAndSum1 = [2, 3, 4]
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
-
-
-
+describe("When cubeArraySum is called", () => {
+  it("return 99 if [2,3,4] is called",() => {
+    //arrange
+    const argument = cubeAndSum1
+    //act
+    const actualResult = cubeArraySum(argument)
+    //assert
+  expect(actualResult).toEqual(99)
+  })
+  it("return 1125 if [0,5,10] is called",() => {
+    //arrange
+    const argument = cubeAndSum2
+    //act
+    const actualResult = cubeArraySum(argument)
+    //assert
+  expect(actualResult).toEqual(1125)
+  })
+})
 
 // b) Create the function that makes the test pass.
 
+//pseudocode
+//create a function takes an array as an argument
+//create variable to take sum of cubed numbers
+//.map array adding numbers to sum variable
+//return sum variable
 
+const cubeArraySum = (array) => {
+  let sumArrayNums = 0
+  array.forEach(number => {
```

Comment:
this would be a good opportunity to see if there's another built-in method that would reduce an array to a single value